### ISSUE TITLE
Specify sex determination variables `RateX`, `RateY`, `RateErrX`, `RateErrY`

### ISSUE THREAD

**Issue by nevrome** [poseidon-framework/poseidon-schema#73]
@TCLamnidis introduced .janno variables in the minotaur workflow that could be universally useful. We could specify them in the next schema release.

Is this a good decision? How exactly would should they be defined?

**Comment by stschiff** [comment]
Hmm, for now I think these are pretty special for our method of sex determination. While they might be of interest for some analyses, their definition is quite specific (relative coverage on X and Y compared to the autosomes). I am not sure that we should cater in the schema for such specific analyses. I think it's fine for now if Minotaur adds these without adding them to the schema.

We could then see, once Minotaur takes a bit more ground and people grow fond of these fields whether to elevate them to the schema. So perhaps we can leave this issue open and give it a `future` tag or something?

**Comment by TCLamnidis** [comment]
I agree with Stephan. I added these variables as they come from the processing and inform the Genetic_Sex field, but the intention is to use these variables to fill the Genetic Sex field prior to publishing to the Minotaur Archive. They can then stick around as they are not costly, but they are indeed quite linked to the specific type of Sex determination analysis done by `sexdeterrmine`, which is but one of many ways to do it.

**Comment by nevrome** [comment]
Ok - good :+1:. I'll set this issue to `wontfix` for now as a reminder. Eventually it can be removed.
